src/seedu/addressbook/ui/TextUi.java

```diff
@@ -129,7 +92,7 @@ public void showResultToUser(CommandResult result) {
         if (resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        formatter.showToUserResult(result.feedbackToUser);
```

Comment:
UI should do the I/O part; formatter should, as the name suggests, do only the formatting.